src/SDK.ts

```diff
@@ -47,24 +47,25 @@ import sessionInitRetryHandler from "./Utils/SessionInitRetryHandler";
 export default class SDK implements ISDK {
   private static defaultRequestTimeoutConfig: RequestTimeoutConfig = {
     getChatConfig: 30000,
-    getLWIDetails: 5000,
+    getLWIDetails: 10000,
     getChatToken: 10000,
-    sessionInit: 5000,
+    sessionInit: 10000,
     sessionClose: 10000,
-    getReconnectableChats: 5000,
-    getReconnectAvailability: 5000,
+    getReconnectableChats: 10000,
```

Comment:
Why are we incrementing the default timeout?

Reply:
With OAuth 2.0 ChannelConnector needs more time to process, and 5 seconds is sometime not enough. Before backend improves its performance, we must increase the timeout here